doc/classes/CanvasItem.xml

```diff
@@ -533,7 +533,17 @@
 			<argument index="0" name="screen_point" type="Vector2">
 			</argument>
 			<description>
-				Assigns [code]screen_point[/code] as this node's new local transform.
+				Transforms [code]screen_point[/code] to this node's local transform.
```

Comment:
```suggestion
				Transforms [code]screen_point[/code] from viewport's coordinates to this node's local coordinates.
```
I'd also rename the argument, because it's misleading. Maybe `canvas_point`.

Reply:
Addressed in #98490